without-tax-task.js

```diff
@@ -0,0 +1,11 @@
+/*Napisz funkcję withoutTax(amount), która zwróci kwotę wynagrodzenia po odjęciu
+podatku dochodowego w wysokości 18 % .Wynik zaokrąglij do pełnych złotych.*/
+
+const taxPercent = 0.82;
```

Comment:
I intentionally suggested 18, not 82, not 0.82 so someone doesn't have to calculate anything in his head.
Also CAPS_LOCK is a convention for constants.